src/main/resources/log4j2.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Configuration status="info">
+    <Appenders>
+        <Console name="LogToConsole" target="SYSTEM_OUT">
+            <PatternLayout pattern="%d{HH:mm:ss.SSS} %-5level %c:%L - %ms%n"/>
+        </Console>
+        <File name="LogToFile" filename="log/app.log">
+            <PatternLayout>
+                <Pattern>%d %p %c:%L - %m%n</Pattern>
+            </PatternLayout>
+        </File>
+    </Appenders>
+    <Loggers>
+        <Logger name="mate.academy" level="info" additivity="false">
+            <AppenderRef ref="LogToFile"/>
+            <AppenderRef ref="LogToConsole"/>
+        </Logger>
+        <Root level="error">
+            <AppenderRef ref="LogToFile"/>
+            <AppenderRef ref="LogToConsole"/>
+        </Root>
+    </Loggers>
+</Configuration>
```

Comment:
add an empty line in the end of file

Reply:
![1](https://user-images.githubusercontent.com/84701063/125027848-a1893100-e08f-11eb-8094-ac1436ec6edc.jpg)

I have an empty line